doc-content/shared-kie-docs/src/main/asciidoc/Workbench/SecurityManagement/business-central-settings-security-policy-con.adoc

```diff
@@ -0,0 +1,25 @@
+[id='business-central-settings-security-policy-con']
+
+= Security policy
```

Comment:
@RishiRajAnand Isn't this just default policy which is applied on the first start of Business Central and then it is stored in the NIOGIT and modified by the user interface? Maybe we should provide steps how to set up default configuration for business central. Wdyt?